Source/Evolve/WarpXEvolve.cpp

```diff
@@ -406,6 +420,118 @@ WarpX::OneStep_nosub (Real cur_time)
     if (warpx_py_afterEsolve) warpx_py_afterEsolve();
 }
 
+void
+WarpX::OneStep_multiJ (amrex::Real cur_time)
+{
+#ifdef WARPX_DIM_RZ
+    amrex::Abort("multi-J algorithm not implemented for RZ geometry");
+#else
+#ifdef WARPX_USE_PSATD
+    if (WarpX::maxwell_solver_id == MaxwellSolverAlgo::PSATD)
+    {
+        // Warnings:
+        // - this does not work with Galilean PSATD
+        //   (the shifts are not properly taken into account when depositing)
+        // - this does not work with PML
```

Comment:
We should abort here if the galilean velocity is different than 0.

Reply:
https://github.com/ECP-WarpX/WarpX/pull/1828/commits/eea274921a4c30ec14fe2f9176d4f81a139c319d: `amrex::Abort` for both cases added in Source/WarpX.cpp.